app/views/posts/index.html.erb

```diff
@@ -1 +1,25 @@
-<h1>This is the first post</h1>
\ No newline at end of file
+<p> This is the list of posts</p>
+
+<span><%= @user.photo %></span>
```

Comment:
- [ ] This should be put in an image tag and also make your page look like the given wireframe